package.json

```
@@ -23,6 +23,6 @@
   },
   "homepage": "https://github.com/Redocly/apidocs-starter#readme",
   "dependencies": {
-    "@redocly/redoc": "^0.93.0"
+    "@redocly/redoc": "^0.93.3"
```

```suggestion
    "@redocly/redoc": "^0.94.2"
```
The `theme` removal was in 0.94.0.